lib/app/app.rb

```diff
@@ -78,14 +76,17 @@ class ConnectorsWebApp < Sinatra::Base
       :connectors_version => settings.version,
       :connectors_repository => settings.repository,
       :connectors_revision => settings.revision,
-      :connector_name => ActiveSupport::Inflector.camelize(settings.http['connector'])
+      :connector_name => settings.connector_name,
+      :display_name => @connector.display_name,
+      :configurable_fields => @connector.configurable_fields,
```

Comment:
Unfortunately this is the one change that makes this PR backwards incompatible without: https://github.com/elastic/ent-search/pull/6504. The nested hash breaks the content source details UI. I'll need to wait until both the Enterprise Search and Kibana PRs are ready to merge.